src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -85,9 +85,14 @@ internal IEnumerable<NodeSearchElement> GetMatchingSearchElements()
         {
             var elements = new List<NodeSearchElement>();
             var inputPortType = PortViewModel.PortModel.GetInputPortType();
+
+            //List of nodes that are skipped temporarily, and will display list of default suggestions instead.
+            var skippedInputPorts = new List<string>() { "var", "object", "string", "bool", "int", "double" };
```

Comment:
how do you all feel about removing string from this list? How many items does it return?

Reply:
`String` currently already return default list, so I guess it could be optional. It is defined by @aparajit-pratap in original AC. That's why @zeusongit added it. If we all agree, I can update the AC
![image](https://user-images.githubusercontent.com/3942418/97052907-e09e4300-154f-11eb-93ec-8e3d78ff1073.png)

Reply:
If no one have strong opinions, let's leave it as it is as the end result is same